packages/Search/cmake/Dependencies.cmake

```diff
@@ -4,4 +4,6 @@ TRIBITS_PACKAGE_DEFINE_DEPENDENCIES(
   Kokkos
   Teuchos
   Tpetra
+  LIB_REQUIRED_TPLS
```

Comment:
Oh yeah. I just grepped boost and I didn't pay attention that we just mention boost in the comment.